Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'faraday'
 gem 'foundation_emails'
 gem 'hiredis'
 gem 'http_accept_language'
-gem 'identity-doc-auth', github: '18F/identity-doc-auth', branch: 'v0.5.0'
+gem 'identity-doc-auth', github: '18F/identity-doc-auth', tag: 'v0.5.1'
```

Comment:
@zachmargolis what was the reason we switched from tag to branch here? I switched it back because that's what we'd been doing but I wanted to check.

Reply:
It should be `tag` , branch was likely a development mistake from me 🤦

Reply:
No problem! I'll merge the gem shortly and tag the new release. :-)